### PR TITLE
pac trails 

### PR DESCRIPTION
**Pull request by huskerman34**
Fixed the wrapper issues. will work on the indetation for the letteing

### CODE REVIEW

**Comment by huskerman34** [comment]
fixing

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<head>
+    <meta charset="utf-8">
+    <title> Pacific Trails Resort:Yurts ></title>
+    <link rel="stylesheet" href="pacifictrails.css">
+    <link rel="icon" href="images/favicon.ico" type="images/x-icon">
+</head>
+<main>
+
+    <body>
```
Comment (mbMosman) [inline activities.html]: The body tag must immediately follow the head tag closing.  Nothing in between.  Always.  

The main tag must be inside of the body.  It should open after the nav is closed.

File: activities.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            <h3>Hiking</h3>
+            <p>Pacific Trails Resort has 5 miles of hiking trails and is adjacent to a state park. Go it alone or join one of our guided hikes.</p>
+            <h3>Kayaking</h3>
+            <p>Ocean kayaks are available for guest use.</p>
+            <h3>Bird Watching</h3>
+            <p>While anytime is a good time for bird watching at Pacific Trails, we offer good time for bird watching trips at sunrise several times a week.</p>
+
+            <link rel="stylesheet" href="pacifictrails.css">
+            <footer>Copyright &copy;2016 Pacific Trails Resort </footer>
+            <a href="yourfirstname@yourlastname.com">Yourfirstname@Yourlastname.com</a>
```
Comment (mbMosman) [inline activities.html]: This contact link needs to be a mailto email link.  Check the textbook for how to do this.  It should also be inside of the footer.

File: activities.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            <h2>Activities at Pacific Trails.</h2>
+            
+            <h3>Hiking</h3>
+            <p>Pacific Trails Resort has 5 miles of hiking trails and is adjacent to a state park. Go it alone or join one of our guided hikes.</p>
+            <h3>Kayaking</h3>
+            <p>Ocean kayaks are available for guest use.</p>
+            <h3>Bird Watching</h3>
+            <p>While anytime is a good time for bird watching at Pacific Trails, we offer good time for bird watching trips at sunrise several times a week.</p>
+
+            <link rel="stylesheet" href="pacifictrails.css">
```
Comment (mbMosman) [inline activities.html]: The stylesheet links go in  the head tag.

File: activities.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <meta charset="utf-8">
+    <title> Pacific Trails Resort:Yurts ></title>
+    <link rel="stylesheet" href="pacifictrails.css">
+    <link rel="icon" href="images/favicon.ico" type="images/x-icon">
+</head>
+<main>
+
+    <body>
+        <div id="wrapper">
+            <h1>Pacific Trails Resort</h1>
```
Comment (mbMosman) [inline activities.html]: The h1 should be inside of a header tag.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            <p><span class="resort"> Pacific Trails Resort</span> offers a specail lodging experience on the California North Coast. Relax in serenity with panoramic views of the Pacific Ocean.</p>
+            <ul>
+                <li>Private Yurts with decks overlooking the ocean</li>
+                <li> Activities lodge with fireplace and gift shop</li>
+                <li>Nightly fine dinning at the Overlook Cafe </li>
+                <li>Heated outdoor pool and whirlpool</li>
+                <li>Guided hiking tours of the redwoods</li>
+            </ul>
+            <div id="contact">
+                Pacific Trails Resort> 12010 Pacific Trails Road Zephyr, CA 95555 888-555-5555
```
Comment (mbMosman) [inline index.html]: You have an extra > in the text.  You also need to use `<br>` tags here in the contact information to break the lines and add the blank line between the address and phone number.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+    <meta charset="utf-8">
+    <header>
+        <title>Pacific Trials Resort</title>
+    </header>
+    <link rel="stylesheet" href="pacifictrails.css">
+    <link rel="icon" href="images/favicon.ico" type="image/x-icon">
+</head>
+
+<main>
```
Comment (mbMosman) [inline index.html]: Same structural errors as above in the activities file.  Make sure to correct in all 3 files.

File: pacifictrails.css [hunk 7]
```diff
@@ -0,0 +1,127 @@
+body {
+    background-image: linear-gradient(to bottom, #ffffff, #90c7e3);
+    background-repeat: no-repeat;
+}
```
Comment (mbMosman) [inline pacifictrails.css]: You are missing some CSS for fonts and font colors.  You should go back through and check the CSS from each of the chapters to ensure nothing is missing for this week.  That means Ch 4, 5, 6, & 7.

File: yurts.html [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    <div id="wrapper">
+
+        <h1>Pacific Trails Resort </h1>
+
+        <nav> <a href="index.html">Home</a> &nbsp; <a href="yurts.html">Yurts</a> &nbsp; <a href="Activities.html">Activities </a> &nbsp; <a href="Reservation.html"> Reservations</a> &nbsp; </nav>
+       
+            <div id="yurthero"></div>
+            <h2>The Yurts at Pacific Trails</h2>
+            <div id="wrapper"></div>
+            <dl>
```
Comment (mbMosman) [inline yurts.html]: The note on the web site indicate that you should use the same h3 & paragraph structure that is found on the activities page here on the Yurts page as well.  https://htc-ccis1301.github.io/fall-2016/morea/cs-pac-trails/ex-cs-pt-part1.html

**Comment by mbMosman** [comment]
I'm closing this for now, but make corrections and reopen a new pull-request when you are complete with the work for this week so that it can be graded.

**Pull request closed without merging**